src/runtime/meter-toast-api.js

```diff
@@ -43,21 +44,27 @@ export class MeterToastApi {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    const iframeArgs = this.activityPorts_.addDefaultArguments({
```

Comment:
As a followup on this, there is probably some central guidance that was followed when creating this initially?  Can we update that guidance?

Reply:
I think Chen copied a previous class that was using feArgs without addDefaultArguments. Here is an example: https://github.com/subscriptions-project/swg-js/blob/c51bdb5af989317b08922d649441f77e91be49e8/src/runtime/login-notification-api.js